tests/test_network.py

```diff
@@ -232,7 +232,7 @@ async def test_network_network_response_completed_event_emitted(
                 "headers": ANY_LIST,
                 "mimeType": "text/html",
                 "bytesReceived": ANY_NUMBER,
-                "headersSize": ANY_NUMBER,
+                "headersSize": AnyOr(338, 355, 360),
```

Comment:
This feels like it can be flaky, as it looks for 3 specific values, maybe just looking for a range between 300 and 400?

Reply:
Updated